dips/views.py

```diff
@@ -661,3 +664,16 @@ def settings(request):
         return redirect("settings")
 
     return render(request, "settings.html", {"form": form})
+
+
+class DIPStoredWebhook(APIView):
+    """
+    Webhook to be called by Archivematica Storage Service when a new
+    DIP is stored.
+    """
+
+    authentication_classes = (authentication.TokenAuthentication,)
+    permission_classes = (permissions.IsAdminUser,)
```

Comment:
To allow any authenticated user, use `permissions.IsAuthenticated`

Reply:
Ah, the DRF permissions module isn't the same thing as Django permissions. We could use them (documented under [DjangoModelPermissions](https://www.django-rest-framework.org/api-guide/permissions/#djangomodelpermissions)) if desired, but it's more complicated.